README.md

```diff
@@ -5,7 +5,9 @@ A tool for formatting Rust code according to style guidelines.
 
 ## How to use
 
-You'll need a pretty up to date version of the **nightly** version of Rust.
+You'll need a pretty up to date version of the **nightly** version of Rust. 
+You will need a default.toml file in the working directory of where you are
```

Comment:
Is this correct? I thought the file had to be in the same directory as the executable.